.github/pull_request_template.md

```diff
@@ -1,13 +1,7 @@
-## Jira Ticket (Mandatory)
+#### Ticket
 
-https://p2pvalidator.atlassian.net/browse/
+[PWN-XXXX](https://p2pvalidator.atlassian.net/browse/XXXX)
```

Comment:
I liked the full hyperlink more

Reply:
Why? 😶

Reply:
The ticket number is redundant here. We use it in the title. 
In the description we just need to switch to Jira quickly, therefore there's only one action from develop: paste the ticket number which he used in the PR title